drivers/volume/portworx/portworx.go

```diff
@@ -146,6 +145,9 @@ const (
 	pxEnableTLS    = "PX_ENABLE_TLS"
 	pxSharedSecret = "PX_SHARED_SECRET"
 
+	//path to root certificate for authenticating tls connections to porx
+	pxCaFile = "PX_CA_CERT_PATH"
```

Comment:
Specifying the cert as a path will require a user to make multiple changes. They'll need to create a secret and then update volume mounts for the volume to mount the secret. Better to take in env variables for the name of the secret and the key in the secret which contains the cert and read the secret in the driver.

Reply:
@disrani-px Thank you, PR was updated.